test/periphery/integration/merkle-campaign/lt/claim/claim.t.sol

```diff
@@ -85,6 +87,32 @@ contract Claim_MerkleLT_Integration_Test is Claim_Integration_Test, MerkleLT_Int
         _;
     }
 
+    function test_Claim_TimestampsInThePast()
```

Comment:
I refactored it to include the following branches in https://github.com/sablier-labs/v2-core/pull/1027:

```tree
    ├── when stream start time zero
    │   ├── it should create a stream with block.timestamp as start time
    │   └── it should emit a {Claim} event
    └── when stream start time not zero
        ├── it should create a stream with `STREAM_START_TIME` as start time
        └── it should emit a {Claim} event
```